app/src/main/java/io/gnosis/safe/notifications/NotificationRepository.kt

```diff
@@ -131,10 +131,13 @@ class NotificationRepository(
         kotlin.runCatching {
```

Comment:
registerSafe(safe: Safe) should be renamed to registerSafe

Reply:
Right now the parameter is needed to to the onSuccess() stuff. And if we do not need the parameter, we can remove registerSafe and call registerOwners instead. And add the two methods from onSuccess for all safes.

Reply:
Is it safe to call `notificationManager.updateNotificationChannelGroupForSafe(safe)` and `safeRepository.saveSafeMeta(SafeMetaData(safe.address, true))` in `onSuccess()` for all safes everytime?

Reply:
For updateNotificationChannelGroupForSafe I think it is safe. But I am not sure about saveSafeMeta.